Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -11,26 +12,39 @@ SCRIPTS = os.path.dirname(os.path.dirname(LOCATION))
 CHERRY_PICKING_RE = re.compile(r'^# You are currently cherry-picking commit (?P<hash>[a-f0-9A-F]+)\.$')
 IDENTIFIER_RE = re.compile(r'(\d+\.)?\d+@\S+')
 PREFER_RADAR = {{ prefer_radar }}
+ENVIRONMENT = os.environ.copy()
 
 sys.path.append(SCRIPTS)
 from webkitpy.common.checkout.diff_parser import DiffParser
 from webkitbugspy import radar
 
 
+cwd = ENVIRONMENT.get('PWD', os.getcwd())
```

Comment:
What motivates allowing `PWD` to override what the system believes the cwd to be?